relay-server/src/services/processor.rs

```diff
@@ -1461,20 +1448,21 @@ impl EnvelopeProcessorService {
     fn normalize_event<G: EventProcessing>(
         &self,
         state: &mut ProcessEnvelopeState<G>,
-    ) -> Result<(), ProcessingError> {
+        mut event_fully_normalized: bool,
+    ) -> Result<Option<bool>, ProcessingError> {
```

Comment:
We should not pass this around as a boolean, it's too easy to have multiple bools mix. Let's make a newtype, it can be as simple as `struct InsertNameHere(pub bool)`.

Reply:
I did not opt for the enum since I want the type to represent the two possible states and then the `Option` refers to whether the calculation should change the value.

I could create a small macro to set the variables automatically, it could make the code nicer but it feels a bit overkill for now.